src/commands/executeFunction/eventGrid/OpenEventGridFileStep.ts

```diff
@@ -0,0 +1,85 @@
+import { AzExtFsExtra, AzureWizardExecuteStep, nonNullProp } from "@microsoft/vscode-azext-utils";
+import * as os from 'os';
+import * as path from "path";
+import * as vscode from 'vscode';
+import { type Progress } from "vscode";
+import { ext } from "../../../extensionVariables";
+import { localize } from "../../../localize";
+import { feedUtils } from "../../../utils/feedUtils";
+import { type ExecuteEventGridFunctionContext } from "./ExecuteEventGridFunctionContext";
+
+export class OpenEventGridFileStep extends AzureWizardExecuteStep<ExecuteEventGridFunctionContext> {
```

Comment:
nit/suggestion: To keep things congruent, it might be better to name this `EventGridFileOpenStep` just so all the steps have the `EventGrid` prefix, making it easier to associate on first glance.

Reply:
Renamed